x/btcstaking/keeper/btc_delegations.go

```diff
@@ -408,14 +407,27 @@ func (k Keeper) validateMultiStakedFPs(ctx sdk.Context, fpBTCPKs []bbn.BIP340Pub
 		if fp.IsSlashed() {
 			return types.ErrFpAlreadySlashed
 		}
+
 		if fp.SecuresBabylonGenesis(ctx) {
-			multiStakedToBabylon = true
+			babylonFpCount++
+
+			if babylonFpCount > 1 {
+				return types.ErrInvalidMultiStakingFPs.Wrap("more than one Babylon finality provider found in the multi-staking selection")
+			}
+		} else {
+			fpConsumerCounters[fp.BsnId]++
```

Comment:
Noob question will this line panick if `fpConsumerCounters[fp.BsnId]` does not exist?